pom.xml

```
@@ -30,6 +30,8 @@
         <google-http-client.version>1.44.2</google-http-client.version>
         <api-helper-java.version>3.0.1</api-helper-java.version>
         <flapdoodle.version>4.16.1</flapdoodle.version>
+        <email-producer-java.version>3.0.2</email-producer-java.version>
+        <version.kafka-clients>2.6.0</version.kafka-clients>
```

Might be better to have .version at the end to be consistent with the others?

```suggestion
        <kafka-clients.version>2.6.0</kafka-clients.version>
```

---

Good spot! Yeah, I agree. I've made the change